check_prime.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+#include<math.h>
+
+main()

```

Comment:
Hi, can you change it to  `int main()` as you are returning 0?

Reply:
Sure